main/features/user/tasks/list/tasks-details.html

```diff
@@ -1,8 +1,8 @@
 <button class="pull-right btn btn-primary"
         id="Button{{editable? 'Release' : 'Take'}}DetailsColumn"
-        ng-click="onTakeReleaseTask()" title="{{editable ? 'Unassign the task and put it back in Available tasks' : 'Assign the task to myself and move it to My tasks.'}}">
+        ng-click="onTakeReleaseTask()" title="{{editable ? 'Unassign the task and put it back in Available tasks' : 'Assign the task to myself and move it to My tasks.' | translate}}">
   <i class="glyphicon" ng-class="{'glyphicon-save':!editable, 'glyphicon-open':editable}"></i>
-  <span class="sr-only">{{editable ? 'Release' : 'Take'}}</span>
+  <span class="sr-only">{{editable ? 'Release' : 'Take' | translate}}</span>
```

Comment:
I don't know :-)

IMHO what will be translated is `Release` or `Take` but I wonder what string will be extracted in pot file.

Let me see